bluemira/geometry/wire.py

```diff
@@ -88,6 +88,10 @@ def _create_shape(self) -> cadapi.apiWire:
 
     def _create_wire(self, *, check_reverse: bool = True):
         wire = cadapi.apiWire(self._get_wires())
+
+        if not cadapi.is_valid(wire):
```

Comment:
this is defined in base so could be self.is_valid

Reply:
I think I tried that and ran into some half initialised weirdness because the wire hasnt been set on _shape at this point